doc/source/architecture.rst

```diff
@@ -0,0 +1,188 @@
+.. _ref_architecture:
+
+=========================
+PyMechanical Architecture
+=========================
+
+PyMechanical is a Python interface to the Mechanical application. Mechanical
+is a polyglot [#f1]_ desktop application whose graphical user interface (GUI)
+can run on both Windows and Linux.
+
+For several years now, Mechanical has had Python scripting capabilities using
+a scripting API. Like many other applications developed by Ansys, scripting
+became heavily used for the purpose of *automation*. A user who  would spend
+time doing the same thing over and over again with the GUI could instead write
+a script to do the same thing, saving time and effort.
+
```

Comment:
To improve efficiency, Mechanical currently enables you to use Python API capabilities to automate repetitive processes.